src/main/java/com/fibermc/essentialcommands/mixin/PlayerManagerMixin.java

```diff
@@ -122,7 +120,7 @@ public void onRespawnPlayer_forResawnLocationOverwrite(
                 Vec3d.ZERO,
                 0,
                 0,
-                false,
+                PositionFlag.ROT,
```

Comment:
since this is a respawn, I think we should just have nothing be "relative" to current position, so I'd just drop this parameter.